src/nitclk/metadata/config.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# This file is generated from NI-TClk API metadata version 255.0.0d0
+config = {
+    'api_version': '255.0.0d0',
+    'c_function_prefix': 'niTClk_',
+    'close_function': None,
+    'context_manager_name': {
+        'abort_function': None,
+        'initiate_function': 'Initiate',
+        'task': 'acquisition'
```

Comment:
NI-TClk can be used for generation too (like in NI-FGEN).
Do you think it matters? Where does this get reflected?

Reply:
Don't know yet. Maybe nowhere? It is called initiate in the tclk API though.

Reply:
I was referring to:
```
        'task': 'acquisition'
 ```

Reply:
Actually, I don't think nitclk can have a context manager, so removed.

Reply:
Is that because there is no "abort" function?

I think there is an expectation we've created for nimi-python users that you can use a context manager, and it would be bad not to support it in nitclk.

```
with nitclk.initiate(session_reference_list)
   # do stuff
# more code after aborting
```

The difference is that we'd need a context manager that on `__exit__` iterates over the `nitclk.SessionReference` objects and calls `abort()` on them. Which means `nitclk.SessionReference` objects need to internally hold on to `nifoo.Session`. But with duck-typing it should all "just work" I believe.

Reply:
We do not do this in other languages. Is that really the right way to program NI-TClk? It doesn't look like it.

Reply:
I have created a detailed issue to track this separately since it's larger than just adding the metadata.
https://github.com/ni/nimi-python/issues/1024